packages/core/logging/core-logging-server-internal/BUILD.bazel

```diff
@@ -0,0 +1,108 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "core-logging-server-internal"
+PKG_REQUIRE_NAME = "@kbn/core-logging-server-internal"
+
+SOURCE_FILES = glob(
+  [
+    "src/**/*.ts",
+  ],
+  exclude = [
+    "**/*.test.*",
+  ],
+)
+
+SRCS = SOURCE_FILES
+
+filegroup(
+  name = "srcs",
+  srcs = SRCS,
+)
+
+NPM_MODULE_EXTRA_FILES = [
+  "package.json",
+]
+
+RUNTIME_DEPS = [
+  "@npm//lodash",
+  "@npm//moment-timezone",
+  "@npm//elastic-apm-node",
+  "//packages/elastic-safer-lodash-set",
+  "//packages/kbn-config-schema",
+]
+
+TYPES_DEPS = [
```

Comment:
As per your step-by-step guide on your approach, can we safely assume that third-party dependencies will export their types as part of their packages?

Our strategy for naming packages really helps with exporting/importing types!

Reply:
> can we safely assume that third-party dependencies will export their types as part of their packages

Some are exporting them directly, and some are using a distinct `@types` package to export them. This is why some `TYPE_DEPS` are using `@npm//@types/package` instead of `@npm/package`